html/php-pre-challenge2/index.php

```diff
@@ -1,9 +1,17 @@
 <?php
 $array = explode(',', $_GET['array']);
 
-// 修正はここから
-for ($i = 0; $i < count($array); $i++) {
+$array_length = count($array);
 
+// 修正はここから
+for ($i = 0; $i < $array_length-1; $i++) {
+  for ($j = 1; $j < $array_length-$i; $j++) {
```

Comment:
```suggestion
for ($i = 0; $i < $array_length - 1; $i++) {
  for ($j = 1; $j < $array_length - $i; $j++) {
```
(修正不要)
要修正とまでは言いませんが、他の行にスタイルを合わせて演算子の左右に半角スペースを入れた方がよいです。エディタの自動整形機能を使いましょう。

Reply:
というか理由がない限り提案されたコードをコピペしたほうがよいです。